Fix SKR mini E2 V2 + BTT_MINI_12864_V1 display

radek8 opened the pull request:
<!--

Submitting a Pull Request

- Please fill out all sections of this form. You can delete the helpful comments.
- Pull Requests without clear information will take longer and may even be rejected.
- We get a high volume of submissions so please be patient during review.

-->

### Description
In the pins_BTT_SKR_MINI_E3_common.h file, there are ill-defined pins that prevent the proper functioning of the BTT_MINI_12864_V1 display
This PR solves the problem. 

<!--

Clearly describe the submitted changes with lots of details. Include images where helpful. Initial reviewers may not be familiar with the subject, so be as thorough as possible. You can use MarkDown syntax to improve readability with bullet lists, code blocks, and so on. PREVIEW and fix up formatting before submitting.

-->

### Requirements
BTT SKR mini E3 V1 - V2 and  BTT mini 12864 V1

<!-- Does this PR require a specific board, LCD, etc.? -->

### Benefits
functional display BTT mini 12864 V1
<!-- What does this PR fix or improve? -->


### Related Issues
https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/686#issuecomment-1264644479
https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/650#issuecomment-1146877576

<!-- Does this PR fix a bug or fulfill a Feature Request? Link related Issues here. -->

radek8 committed:
BTT_MINI_12864_V1

thisiskeithb commented on Marlin/Configuration.h:
```diff
@@ -2823,6 +2823,8 @@
 //
 // BigTreeTech Mini 12864 V1.0 is an alias for FYSETC_MINI_12864_2_1. Type A/B. NeoPixel RGB Backlight.
+// If the display lights up but does not show characters, edit the parameter in the file: Marlin\src\lcd\dogm\marlinui_DOGM.cpp
+// change parameter: void MarlinUI::_set_contrast() { u8g.setContrast(contrast); } change to: void MarlinUI::_set_contrast() { u8g.setContrast(255); }
```
Resetting EEPROM should set the contrast correctly, so you shouldn’t need to edit this file.

radek8 replied:
I don't think the comment will hurt anyone. But I will remove it well. It works as you say.

thisiskeithb replied:
We'll undoubtedly get a support request from someone trying to modify core Marlin files, so resetting EEPROM is prefered.

Vendors like Fysetc, MKS, and BTT have that blurb in their LCD instruction manual to force the contrast to the max setting which skips the EEPROM reset step.

radek8 replied:
Ok

radek8 committed:
Update Configuration.h

thinkyhead committed:
run pinsformat.js

thinkyhead committed:
Remove "SKR_MINI_E3_V2"

thinkyhead committed:
just remove invalid pins

thinkyhead commented on Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_common.h:
```diff
@@ -285,24 +278,21 @@
      */
 
-    #define LCD_PINS_RS              EXP1_03_PIN    // CS
-    #define LCD_PINS_ENABLE                 PA3     // MOSI
```
Since these pins are apparently meaningless, we can just remove them.

radek8 replied:
ok

thinkyhead commented:
I've applied a little extra cleanup, and it looks like nothing is broken. If it all looks good then this can be merged today.

radek8 commented:
Thank you

radek8 committed:
Update pins_BTT_SKR_MINI_E3_common.h

wiring repair according to:
https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/686#issuecomment-1296545443
